src/styles/header.scss

```diff
@@ -0,0 +1,30 @@
+.header {
+  box-sizing: border-box;
+  background-size: cover;
+  background-position: center;
+
+  display: flex;
+  flex-direction: column;
+
+  &__bottom {
```

Comment:
BEM elements should be inside the parent block
In your case it's a separate block, `catalog` or `content` for example

Reply:
i dont actually understand, i have neither catalog block nor content, can u please describe in more detail